set.mm

```diff
@@ -160005,8 +160006,7 @@ at index N (modulo the length of the word) of the original word.
 
   $( The symbol at index ((n-N)-1) of a word of length n (not 0) cyclically
      shifted by N positions is the symbol at index (n-1) of the original word.
-     (Contributed by AV, 23-May-2018.)  (Revised by AV, 21-May-2018.)  (Revised
-     by AV, 30-Oct-2018.) $)
+     (Contributed by AV, 23-May-2018.)  (Revised by AV, 30-Oct-2018.) $)
```

Comment:
@avekens I took a look around at the theorems here and made my best guess at the true history, although I didn't do a git blame. Do you have any extra insight into how the dates got mixed up here?

Reply:
Aha, git blame reveals the truth: the date got mangled in 48c50b251 . Will fix.